### ISSUE TITLE
`wsl --version` & WSLg not working on WSL2 1.0 on Win10 22H2; empty /mnt/wsl; missing /tmp/.X11

### ISSUE THREAD

**Issue by bentolor** [microsoft/WSL#9205]
### Version

Microsoft Windows [Version 10.0.19045.2251]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.102.1

### Distro Version

Ubuntu-22.04,Ubuntu

### Repro Steps

1. Having WSL2 installed & used pre v.1.0
2. Receive Win 22H2 update + WSL2 v1.0 store version
3. Run `wsl --update`
4. Have WSL2 feature & all WSL components deinstalled; reinstall WSL2 store version. Reinstall by software request a) WSL2 feature b) WSL2 Update patch
5. Start a freshly created & downloaded Ubuntu 22.04.1; install `gedit` on it

### Expected Behavior

1. `wsl --version` should show version
2. Launching a WSL2 Ubuntu instance should allow launching & displaying graphical programs like `gedit`

### Actual Behavior

1. `wsl --version` shows a short helpscreen (w/ uninstalled win feature) or long helpscreen in installed state
2. Launching of `gedit` inside WSL not possible
3. There is no `/tmp/.X11` mount or `/mnt/wslg` 
4. There is an empty `/mnt/wsl` and a populated `/mnt/c` is there

I'm totally lost. I also tried disabling `systemd`, did `sfc` and `dism`, uninstalled/reinstalled, tried creating a firewall exception, went through many things mentioned here or there (i.e. issue 8888), went through many hints on wslg issue tracker. 

Can anybody point me how to further diagnose my issue?

### Diagnostic Logs

[WslLogs-2022-11-24_15-39-15.zip](https://github.com/microsoft/WSL/files/10085519/WslLogs-2022-11-24_15-39-15.zip)

**Comment by MrROBUST** [comment]
Duplicate of #9157

tldr: Check additional updates for Windows, you need KB5020030.

**Comment by bentolor** [comment]
**Oh my dear! That's it. gedit in Win 10! AWESOME!** Thanks @MrROBUST! 

I already [visited that page](https://support.microsoft.com/en-gb/topic/november-15-2022-kb5020030-os-builds-19042-2311-19043-2311-19044-2311-and-19045-2311-preview-237a9048-f853-4e29-a3a2-62efdbea95e2) today and somewhat misinterpreted that I should go for _KB5015684_ instead. 

As I'm on organize-managed installation I did not see _KB5020030_. Just found the [manual download page for KB5020030](https://www.catalog.update.microsoft.com/Search.aspx?q=KB5020030) and currently installing…
